### PR TITLE
nieve slayer list

### PR DESCRIPTION
**Pull request by wyattos**

### CODE REVIEW

File: src/monsters/nieve/AbyssalDemon.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+	.tertiary(25, 'Ensouled abyssal head')
+	.tertiary(128, 'Clue scroll (hard)')
+	.tertiary(1200, 'Clue scroll (elite)')
+	.tertiary(6000, 'Abyssal head')
+
+	/* Catacombs tertiary */
+	.tertiary(233, 'Ancient shard')
+	.tertiary(350, 'Dark totem base')
+	.tertiary(350, 'Dark totem middle')
+	.tertiary(350, 'Dark totem top');
```
Comment (gc) [inline src/monsters/nieve/AbyssalDemon.ts]: new line here

File: src/monsters/nieve/Drake.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+	.add('Grimy avantoe', [1, 3], 10)
+	.add('Grimy kwuarm', [1, 3], 10)
+	.add('Grimy ranarr weed', [1, 3], 8)
+	.add('Grimy snapdragon', [1, 3], 8)
+	.add('Grimy cadantine', [1, 3], 8)
+	.add('Grimy dwarf weed', [1, 3], 8)
+	.add('Grimy lantadyme', [1, 3], 6)
+	.add('Grimy torstol', [1, 3], 6);
+
+export const AnkouTable = new LootTable()
```
Comment (gc) [inline src/monsters/nieve/Drake.ts]: wrong name

File: src/monsters/nieve/GreaterDemon.ts [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+	/* Tertiary */
+	.tertiary(40, 'Ensouled demon head')
+	.tertiary(128, 'Clue scroll (hard)')
+
+	/* Catacombs tertiary */
+	.tertiary(246, 'Ancient shard')
+	.tertiary(370, 'Dark totem base')
+	.tertiary(370, 'Dark totem middle')
+	.tertiary(370, 'Dark totem top');
+export default new SimpleMonster({
```
Comment (gc) [inline src/monsters/nieve/GreaterDemon.ts]: new line here

File: src/monsters/nieve/Hellhound.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import LootTable from '../../structures/LootTable';
+import SimpleMonster from '../../structures/SimpleMonster';
+
+export const HellhoundTable = new LootTable()
+	.every('Bones')
+
+	/* Other */
+	.oneIn(32768, 'Smouldering stone')
```
Comment (gc) [inline src/monsters/nieve/Hellhound.ts]: use `32_768` for readability

File: test/Clues.ts [hunk 5]
```diff
@@ -17,8 +17,10 @@ test('Beginner Clues', async (test): Promise<void> => {
 		'Steel platebody': 22.36,
 		'Wizard hat': 22.36
 	};
-
-	const number = 500_000;
+	/*
+		const number = 500_000;
```
Comment (gc) [inline test/Clues.ts]: you need to undo these changes to the clue test file

File: src/monsters/subtables/CommonSeedDropTable.ts [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+	.addItem('Strawberry seed', 1, 131)
+	.addItem('Marrentill seed', 1, 125)
+	.addItem('Jangerberry seed', 1, 92)
+	.addItem('Tarromin seed', 1, 85)
+	.addItem('Wildblood seed', 1, 83)
+	.addItem('Watermelon seed', 1, 63)
+	.addItem('Harralander seed', 1, 56)
+	.addItem('Snape grass seed', 1, 40)
+	.addItem('Ranarr seed', 1, 39)
+	.addItem('Ranarr seed', 1, 34)
```
Comment (Alexsuperfly) [inline src/monsters/subtables/CommonSeedDropTable.ts]:
```suggestion
	.addItem('Whiteberry seed', 1, 34)
```

File: test/Clues.ts [hunk 7]
```diff
@@ -17,7 +17,6 @@ test('Beginner Clues', async (test): Promise<void> => {
 		'Steel platebody': 22.36,
 		'Wizard hat': 22.36
 	};
-
```
Comment (Alexsuperfly) [inline test/Clues.ts]: revert

File: test/Clues.ts [hunk 8]
```diff
@@ -78,6 +77,7 @@ test('Medium Clues', async (test): Promise<void> => {
 	};
 
 	const number = 1_000_000;
+
```
Comment (Alexsuperfly) [inline test/Clues.ts]:
```suggestion
```

File: test/Clues.ts [hunk 9]
```diff
@@ -112,6 +112,7 @@ test('Hard Clues', async (test): Promise<void> => {
 	};
 
 	const number = 20_000_000;
+
```
Comment (Alexsuperfly) [inline test/Clues.ts]:
```suggestion
```

File: test/Clues.ts [hunk 10]
```diff
@@ -138,6 +139,7 @@ test('Elite Clues', async (test): Promise<void> => {
 	};
 
 	const number = 25_000_000;
+
```
Comment (Alexsuperfly) [inline test/Clues.ts]:
```suggestion
```

File: test/Clues.ts [hunk 11]
```diff
@@ -172,6 +174,7 @@ test('Master Clues', async (test): Promise<void> => {
...
```
Comment (Alexsuperfly) [inline test/Clues.ts]:
```suggestion
```

File: src/monsters/nieve/AdamantDragon.ts [hunk 12]
```diff
@@ -8,36 +8,36 @@ export const AdamantDragonTable = new LootTable()
...
 
 	/* Materials */
-	.addItem('Adamant bolts(unf)', [20, 40], 22)
-	.addItem('Adamant javelin heads', [40, 50], 16)
-	.addItem('Diamond', [1, 3], 14)
-	.addItem('Dragon javelin heads', [20, 30], 14)
-	.addItem('Adamantite ore', [8, 20], 12)
-	.addItem('Adamantite bar', [5, 35], 8)
-	.addItem('Dragon bolts (unf)', [15, 20], 2)
+	.addItem('Adamant bolts(unf)', [20, 40], 1)
```
Comment (Alexsuperfly) [inline src/monsters/nieve/AdamantDragon.ts]:
```suggestion
	.addItem('Adamant bolts(unf)', [20, 40], 11)
```

File: src/monsters/nieve/Bloodveld.ts [hunk 13]
```diff
@@ -32,9 +32,31 @@ export const BloodveldTable = new LootTable()
 	.add('Coins', 450, 1)
 
 	/* Other */
-	.add('Bones', 1, 10)
-	.add('Big bones', 1, 7)
-	.add('Big bones', 3, 3)
+
+	.addItem(
```
Comment (Alexsuperfly) [inline src/monsters/nieve/Bloodveld.ts]: remove this

File: src/monsters/subtables/WyvernHerbTable.ts [hunk 14]
```diff
@@ -0,0 +1,11 @@
+import LootTable from '../../structures/LootTable';
+
+const WyvernHerbTable = new LootTable()
+	.addItem('Grimy kwuarm', 2, 298)
```
Comment (Alexsuperfly) [inline src/monsters/subtables/WyvernHerbTable.ts]:
```suggestion
	.addItem('Grimy kwuarm', 2, 260)
```

File: src/monsters/subtables/WyvernHerbTable.ts [hunk 15]
```diff
@@ -0,0 +1,11 @@
...
+	.addItem('Grimy cadantine', 2, 187)
```
Comment (Alexsuperfly) [inline src/monsters/subtables/WyvernHerbTable.ts]:
```suggestion
	.addItem('Grimy cadantine', 2, 208)
```

File: src/monsters/subtables/WyvernHerbTable.ts [hunk 16]
```diff
@@ -0,0 +1,11 @@
...
+	.addItem('Grimy dwarf weed', 2, 187)
```
Comment (Alexsuperfly) [inline src/monsters/subtables/WyvernHerbTable.ts]:
```suggestion
	.addItem('Grimy dwarf weed', 2, 208)
```

File: src/monsters/subtables/WyvernHerbTable.ts [hunk 17]
```diff
@@ -0,0 +1,11 @@
...
+	.addItem('Grimy ranarr weed', 1, 166)
```
Comment (Alexsuperfly) [inline src/monsters/subtables/WyvernHerbTable.ts]:
```suggestion
	.addItem('Grimy ranarr weed', 1, 185)
```

File: src/monsters/subtables/WyvernHerbTable.ts [hunk 18]
```diff
@@ -0,0 +1,11 @@
...
+	.addItem('Grimy torstol', 1, 166)
```
Comment (Alexsuperfly) [inline src/monsters/subtables/WyvernHerbTable.ts]:
```suggestion
	.addItem('Grimy torstol', 1, 185)
```

File: src/monsters/subtables/WyvernHerbTable.ts [hunk 19]
```diff
@@ -0,0 +1,11 @@
...
+	.addItem('Grimy lantadyme', 2, 71);
```
Comment (Alexsuperfly) [inline src/monsters/subtables/WyvernHerbTable.ts]:
```suggestion
	.addItem('Grimy lantadyme', 2, 155);
```

File: src/monsters/subtables/RareSeedTable.ts [hunk 20]
```diff
@@ -12,7 +12,7 @@ const RareSeedTable = new LootTable()
 	.addItem('Snapdragon seed', 1, 10)
 	.addItem('Cadantine seed', 1, 7)
 	.addItem('Lantadyme seed', 1, 5)
-	.addItem('Snape grass seed', 3, 4)
+	.addItem('Snape grass seed', 1, 4)
```
Comment (Alexsuperfly) [inline src/monsters/subtables/RareSeedTable.ts]: revert this
